src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -978,7 +827,8 @@ export default {
 	position: relative;
 
 	&:hover .normal-message-body,
-	&:hover .combined-system {
+	&:hover .combined-system,
+	&--hovered  {
```

Comment:
```suggestion
	&--hovered .normal-message-body {
```
Like this probably, for a style consistency